browser/src/slideshow/LayerDrawing.ts

```diff
@@ -541,6 +570,15 @@ class LayerDrawing {
 			type: info.type,
 			content: info.content,
 		};
+		var img_ = (window as any).fzstd.decompress(img.rawData);
```

Comment:
I must be making some incorrect call to decompress. 

function returns uInt8Array of correct expected length but all the elements in it are 0.

Reply:
What's needed is probably similar to impl_encodeSubBufferToPNG which checks the tileMode and does a channel swap if its the mode that needs that, which is the mode your core build is in.